examples/src/main/java/com/yugabyte/examples/UniformLoadBalanceExample.java

```diff
@@ -108,15 +114,14 @@ private static void testUsingHikariPool(String poolName, String lbpropvalue, Str
       pauseApp(".jdbc_example_app_checker3");
```

Comment:
Shouldn't the pauses be inside the interactive flag checks. I see only printlns.

Reply:
this pauseApp() method is basically for communicating between the shell script and java app, so the java app pauses(infinite while loop) till the script tells the app to continue after some executions like adding a node, or deleting node in the cluster.
the string passed here is for the script to check whether this file name exists or not.

Reply:
OK. Please add some comment to indicate what are you pausing for. I believe interactive and verbose are not same. Printlns should be there irrespective but sleep should be there inside interactive and should be awakened on user's input. Pleaes check for this.

Reply:
Yes INTERACTIVE and VERBOSE are different.
Yes, the interactive will awake on some input (Press "ENTER") from the user's end(have checked it already). 
Just adding the comments for the pauseApp() method calls, please check again in 1-2 minutes.